package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "0x",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Can you revert this? We'll bump it on releaze

Reply:
Sure, i got it!